examples/structured_data/classification_with_grn_and_vsn.py

```diff
@@ -2,7 +2,7 @@
 Title: Classification with Gated Residual and Variable Selection Networks
 Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
 Date created: 2021/02/10
-Last modified: 2021/02/10
+Last modified: 2025/01/03
```

Comment:
This seems unrelated. Can you split these changes of this other examples in a separate PR?